src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -43,17 +43,41 @@ const useResolveStep = () => {
     currentLocation: state.session.currentLocation,
   }));
 
+  const mergeCycleCountItems = (items) => {
+    const duplicatedItems = _.groupBy(items,
+      (item) => `${item.binLocation?.id}-${item?.inventoryItem?.lotNumber}`);
+    return Object.values(duplicatedItems).map((itemsToMerge) => {
+      const maxCountIndex = _.maxBy(itemsToMerge, 'countIndex').countIndex;
+      const itemFromCount = _.find(itemsToMerge, (item) => item.countIndex === maxCountIndex - 1);
+      const itemFromResolve = _.find(itemsToMerge, (item) => item.countIndex === maxCountIndex);
+      return itemFromCount ? {
```

Comment:
I think you always will have `itemFromCount`, see effectively you won't land in the `else` in this ternatry operator